compose/install.md

```diff
@@ -34,28 +34,59 @@ To install Compose, do the following:
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
     ```
 
-    For example, to download Compose version 1.12.0, the command is:
+    For example, to download Compose version 1.13.0, the command is:
```

Comment:
You could add a variable in the frontmatter of the topic like `compose-ver: 1.13.0` and then do `{{compose-ver}}` so you don't hvae to remember to update it multiple places in the file?